BlazarService/src/main/java/com/hubspot/blazar/util/BuildLauncher.java

```diff
@@ -119,6 +134,17 @@ private String buildCommand(Module module) {
     return modulesToBuild.contains(module.getName()) ? "--safe_mode" : "--dry_run";
   }
 
+  private BuildConfig configAtSha(String name, String sha, GitInfo gitInfo) throws IOException {
+    LOG.info("Trying to fetch current config at sha: {}", sha);
+    GitHub gitHub = gitHubFor(gitInfo);
+    GHRepository repository = gitHub.getRepository(gitInfo.getFullRepositoryName());
```

Comment:
Would be nice to reuse the `GHRepository` from before to save a call to GitHub, maybe fetch it outside these functions and pass it into `currentSha` and `configAtSha`?